app/javascript/app/pages/country-compare/country-compare-component.jsx

```diff
@@ -57,11 +57,17 @@ const CountryCompare = ({ route, anchorLinks }) => {
           </Desktop>
           {route.sections &&
             route.sections.length > 0 &&
-            route.sections.map(section => (
-              <section key={section.hash} id={section.hash}>
-                {!!section.component && <section.component />}
-              </section>
-            ))}
+            route.sections
+              .filter(s => !(isContained && s.hash === 'climate-vulnerability'))
```

Comment:
Instead of doing the filtering like this by the section hash, could we add it to the routes config a flag like "includeInContained", here:

`app/javascript/app/routes/app-routes/country-compare-sections/country-compare-sections.js`

This way if we need to add more sections and need to exclude them we can use that config option more easily instead of changing that direct comparison.